app/forms/document_type_upload_form.rb

```diff
@@ -1,4 +1,7 @@
 class DocumentTypeUploadForm < QuestionsForm
+  def self.scoped_attributes
+    @@scoped_attributes ||= {}
+  end
```

Comment:
This was another unfortunate addition as a consequence of the RequireDocumentUploadForm inheriting from the DocumentTypeUploadForm and wanting to have the scoped_attributes set by `set_attribute_for` to be shared. Again just sticking with the DocumentTypeUploadForm is probably the way to go.